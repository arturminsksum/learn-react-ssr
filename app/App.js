import React, { Fragment } from 'react';
import Header from './components/header';
import Main from './components/main';
import Footer from './components/footer';
import VisibleArticles from './containers/visible-articles';

const App = () => (
  <Fragment>
    <Header />
    <VisibleArticles />
    <Footer />
  </Fragment>
);

export default App;
