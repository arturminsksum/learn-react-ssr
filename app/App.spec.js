import React from 'react';
import App from './App';

describe('App', () => {
  it('should render App correctly', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
