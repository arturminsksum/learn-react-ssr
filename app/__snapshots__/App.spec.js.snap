// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render App correctly 1`] = `
<React.Fragment>
  <withRouter(Connect(Header)) />
  <div
    className="container"
  >
    <br />
    <Switch>
      <Route
        component={[Function]}
        exact={true}
        path="/"
      />
      <Route
        component={[Function]}
        path="/add"
      />
      <Route
        component={[Function]}
        path="/login"
      />
      <Route
        component={[Function]}
        path="/signup"
      />
      <Redirect
        push={false}
        to="/"
      />
    </Switch>
    <br />
  </div>
  <Component />
</React.Fragment>
`;
